datafusion/common/src/dfschema.rs

```diff
@@ -2106,7 +2118,7 @@ mod tests {
                 Field::new(
                     "timestamp_field",
                     DataType::Timestamp(
-                        arrow::datatypes::TimeUnit::Microsecond,
+                        TimeUnit::Microsecond,
```

Comment:
Seems like the convention here is to not import the type, but to use the qualified path `                       arrow::datatypes::TimeUnit::Microsecond` - maybe remove that import on line 1335?